src/main/scala/scorex/core/NodeViewHolderSharedMessages.scala

```diff
@@ -0,0 +1,39 @@
+package scorex.core
+
+import scorex.core.NodeViewHolder.NodeViewHolderEvent
+import scorex.core.consensus.{HistoryReader, SyncInfo}
+import scorex.core.transaction.box.proposition.Proposition
+import scorex.core.transaction.{MempoolReader, Transaction}
+import scorex.core.transaction.state.StateReader
+
+trait NodeViewHolderSharedMessages {
```

Comment:
shared with whom?

Reply:
In the case of `NodeViewHolderSharedMessages`, it is shared between `NodeViewSynchronizer`, `LocalInterface` and `NodeViewHolder`

Reply:
Maybe rename it to `NodeViewLocalInterfaceSharedMessages` then...

As discussed, these shared messages traits/objects are a temporary solution.